### DISCUSSION TITLE
Recommended way to build additional Python packages?

### DISCUSSION THREAD

**Discussion by bhilton** [indygreg/python-build-standalone#367] [Q&A]
I would like to build and package a set of additional Python packages to go along with the "standalone" Python tarball, such that it benefits from the "standalone" goodness. Static linking where possible, bundling shared libs where necessary, using patchelf to ensure those shared libs are used, etc. Does this community have a turnkey solution for this yet? It would seem that the docker images that are used to build the standalone Python would need to be reused in this later build stage, or perhaps even modified with additional dependencies. I looked through the docs and discussions but couldn't find this topic referenced. Thanks in advance!

**Comment by zanieb** [comment]
I don't know of anyone doing this. Sounds cool though!
